src/components/SingleReview.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from "react";
+import { getSingleReview } from "./api";
+import { v4 as uuidv4 } from "uuid";
+import { useParams } from "react-router-dom";
+
+
+export default function SingleReview({ isLoading, setIsLoading }) {
+
+  const [review, setReview] = useState({
+    "reviews": {
```

Comment:
I think this should just be an empty object. It'll make your code loads DRY-er because you won't have to keep writing our review.reviews later on in your return statement.

Reply:
Please also add in an isError pattern for if a user goes to an endpoint for an id that doesn't exist.